app/main.py

```diff
@@ -1,25 +1,68 @@
 from abc import ABC
+from pyclbr import Class
+from typing import Any
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: Any, name: str) -> None:
+        self.protected_name = f"_{name}"
+
+    def __get__(self, instance: Any, owner: Any) -> None:
```

Comment:
The `__get__` method should return the value of the attribute, but it currently returns `None`. Change the return type to `Any` to reflect that it should return the attribute value.